time.php

```diff
@@ -0,0 +1,10 @@
+<?php
+  $data = [];
+  try {
+      $data = file_get_contents("https://digitalservices.sfgov.org/servertime.php");
+  } catch (Exception $ex) {
+      echo $ex;
```

Comment:
I think it'd be better to log the exception here and either:

* Return structured JSON with the error message
* Return a `500`

Reply:
yeah, good point.  did it in a hurry, forgot all about it

Reply:
addressed in commit: 21899593ee9b83eed4efe8465d58901db7031741

Reply:
👍